app/views/menus/_admin_tabs.haml

```diff
@@ -3,8 +3,13 @@
     - if can_edit_users?
       %li.nav-item{role: "presentation", class: ('active' if current_page?(admin_users_path))}
         = link_to 'Users', admin_users_path, class: 'nav-link'
+    - if can_edit_roles?
```

Comment:
I don't think this is necessary,  you should be able to assign roles even if you can't edit them.  It should use `can_edit_users?`

Reply:
This tab takes you to the roles management page, and the page is guarded by `before_action :require_can_edit_roles!`, so I was hiding the tab when you can't open it.

Reply:
my mistake, that's correct.